### PR TITLE
fix: change api to include optional statuses

### PR DESCRIPTION
**Pull request by PhilippFr**
Signed-off-by: PhilippFruh <philipp.fruh@pm.me>

Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: src/main/java/com/vi/appointmentservice/api/controller/AskerController.java [hunk 1]
```diff
@@ -24,8 +25,21 @@ public class AskerController implements AskersApi {
   private final @NonNull AskerFacade askerFacade;
 
   @Override
-  public ResponseEntity<List<CalcomBooking>> getAllBookingsOfAsker(String askerId) {
-    return new ResponseEntity<>(askerFacade.getAllBookingsOfAskerHandler(askerId), HttpStatus.OK);
+  public ResponseEntity<List<CalcomBooking>> getAllBookingsOfAsker(String askerId, String status) {
```
Comment (adnanalicic) [inline src/main/java/com/vi/appointmentservice/api/controller/AskerController.java:28]: advice seeker can only see upcoming appointments. we this implementation, we have to change the frontend and we need to add more tests, even though we don't need this functionality now.

File: src/main/java/com/vi/appointmentservice/api/facade/AskerFacade.java [hunk 2]
```diff
@@ -15,25 +14,56 @@
...
+          .findByAskerId(askerId);
+      List<CalcomBooking> bookings = calComBookingService.getAskerCancelledBookings(bookingIds.stream().map(CalcomBookingToAsker::getCalcomBookingId).collect(
+          Collectors.toList()));
+      return bookings;
+    } else {
+      return new ArrayList<>();
+    }
+  }
+
+  public List<CalcomBooking> getAskerExpiredBookings(String askerId) {
```
Comment (adnanalicic) [inline src/main/java/com/vi/appointmentservice/api/facade/AskerFacade.java:57]: same again here.

**Review by adnanalicic — CHANGES_REQUESTED** [review]

File: src/main/java/com/vi/appointmentservice/api/controller/AskerController.java (hunk 1 above)
Comment (PhilippFr) [reply src/main/java/com/vi/appointmentservice/api/controller/AskerController.java:28]: Okay, I missunderstood when you said the asker API should work like the consultant API.

Comment (PhilippFr) [reply src/main/java/com/vi/appointmentservice/api/controller/AskerController.java]: @adnanalicic Please re-review

**Review by adnanalicic — APPROVED** [review]

**Pull request merged**